pom.xml

```diff
@@ -3,7 +3,7 @@
     <groupId>com.github.javafaker</groupId>
     <artifactId>javafaker</artifactId>
     <packaging>jar</packaging>
-    <version>0.17.3-SNAPSHOT</version>
+    <version>0.17.4-SNAPSHOT</version>
```

Comment:
No sure why you need to bump this version

Reply:
hmm, a new code version needs a new pom version right? but if you say that don't need, i remove this line.

Reply:
Yes but only when one does a release